policies/allowed-repos/src.rego

```diff
@@ -0,0 +1,20 @@
+# @title Images must come from allowed repositories only
```

Comment:
please create policies directly in their own file instead of folders with one file? Why do you do such things? I hope you start wondering about such things: why would I anticipate B$ and not KISS?

Reply:
Konstraint library expects this structure in order to generate the manifests.